deploy/examples/operator.yaml

```diff
@@ -28,6 +28,8 @@ data:
   # Allow using loop devices for osds in test clusters.
   ROOK_CEPH_ALLOW_LOOP_DEVICES: "false"
 
+  # Enable CSI Operator
+  ROOK_USE_CSI_OPERATOR: "false"
```

Comment:
After this has stabilized a bit and your manual testing is looking good, let's try temporarily enabling this to `true`. Then if the CI passes it will be a good indication that the new csi operator is working end to end. Then in a separate PR we can enable the csi operator in some subset of integration tests.

Reply:
will be done later in the PR